images/monitoring/script.sh

```diff
@@ -0,0 +1,73 @@
+
```

Comment:
You might want to add the Shebang tag here

Reply:
The script is not being used, it is there for reference purposes (It's not complete but might be used in future)